README.md

```diff
@@ -133,13 +128,15 @@ There are some active contribution projects underway right now:
 How Do I Interact With the Community?
 =====================================
 
-* Join the Eclipse OMR developer community [mailing list](https://accounts.eclipse.org/mailing-list/omr-dev).
-The community typically uses this list for project announcements and administrative
-discussions amongst committers.  Questions are welcome here as well.
+We are operating under the [Eclipse Code of Conduct][coc] to promote fairness, openness, and inclusion.
```

Comment:
"We operate under" is stronger . This one's a nit, but since I'm reviewing....:)

Reply:
Thanks for the suggestion, I agree it is stronger